mapper/mapper.go

```diff
@@ -0,0 +1,82 @@
+package mapper
+
+import (
+	"encoding/json"
+	"errors"
+	log "github.com/sirupsen/logrus"
+)
+
+const PREDICATE_IS_CLASSIFIED_BY = "http://www.ft.com/ontology/classification/isClassifiedBy"
+const PREDICATE_IS_PRIMARILY_CLASSIFIED_BY = "http://www.ft.com/ontology/classification/isPrimarilyClassifiedBy"
+const PREDICATE_MAJOR_MENTIONS = "http://www.ft.com/ontology/annotation/majorMentions"
+const PREDICATE_ABOUT = "http://www.ft.com/ontology/annotation/about"
+const CONCEPT_TYPE_TOPIC = "http://www.ft.com/ontology/Topic"
+const CONCEPT_TYPE_LOCATION = "http://www.ft.com/ontology/Location"
+const CONCEPT_TYPE_SPECIAL_REPORT = "http://www.ft.com/ontology/SpecialReport"
+const CONCEPT_TYPE_SUBJECT = "http://www.ft.com/ontology/Subject"
+
+func ConvertPredicates(body []byte) ([]byte, error) {
+	originalAnnotations := make([]map[string]interface{}, 0)
+	convertedAnnotations := make([]map[string]interface{}, 0)
+	err := json.Unmarshal(body, &originalAnnotations)
+	if err != nil {
+		log.Fatal("Could not unmarshall json body", err)
+	}
+
+	for i := 0; i < len(originalAnnotations); i++ {
+		annoMap := originalAnnotations[i]
+		pred, ok := annoMap["predicate"]
+		if !ok {
+			log.Info("no predicate")
```

Comment:
maybe " no predicate supplied  for annotation" , as mentioned annotations-rw-neo4j makes no predicate into MENTIONS , do we want that logic?